docs/runbook-trec2018-anserini.md

```diff
@@ -1,4 +1,16 @@
-# Anserini: TREC 2018 Runbook
+# TREC 2018 Runbook: "Anserini" Group
+
+This is the runbook for TREC 2018 submissions by the "Anserini" group. Note that Anserini (the system) was used by another group ("h2oloo") for a completely different set of runs.
+
+In total, the Anserini group participated in three tracks:
+
++ Common Core Track
++ CENTRE Track
```

Comment:
I think you have put CENTRE to the bottom?

Reply:
fixed.